static/src/javascripts/projects/commercial/modules/third-party-tags/plista-outbrain-renderer.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import externalContentContainerStr from 'raw-loader!common/views/commercial/external-content.html';
+import { plista } from 'commercial/modules/third-party-tags/plista';
+import { initOutbrain } from 'commercial/modules/third-party-tags/outbrain';
+import template from 'lodash/template';
+import config from '../../../../lib/config';
+import fastdom from '../../../../lib/fastdom-promise';
+import $ from '../../../../lib/$';
+
+export class PlistaOutbrainRenderer {
```

Comment:
I don't think it's necessary to use an ES6 class here. We tend to use classes only on occasions where we need to create multiple instances of an Object which will have a common interface and properties and which may or may not be intended to be extensible. However the code here that initialises `Outbrain` or `Plista` is only executed once. Instead this module could just export a single function `init` (formerley known as `render`) and all the functions which were previously properties of `PlistaOutbrainRenderer` (`findAnchor`/`renderWidget`) can be moved to the scope of the module as shown below:

```JS
const findAnchor = () => { ... };
const renderWidget = () => { ... };

export const init = () => { ... };
```

This would match the pattern we follow elsewhere.

It'll mean your tests need updating as we won't be able to spy on `renderWidget` as it'll be a perivate function of the module, as an alternative we could validate against whether the `Outbrain` or `Plista` init function has been called.